plugins/jira/commands/generate-feature-updates.md

```
@@ -0,0 +1,463 @@
+
+If validation fails (missing charset, mojibake detected), regenerate the HTML using the template above before proceeding. **Do not serve HTML that fails this check.**
+
+5. Start a local HTTP server in the background:
+
+```bash
+python3 -m http.server 8787 --directory /tmp --bind 127.0.0.1
+```
+
+6. Display the URL to the user:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)